### PR TITLE
Adding Robotics-Nav2-SLAM-Example to "Getting Started" list

### PR DESCRIPTION
**Pull request by mrpropellers**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,14 +14,20 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
...
 
 ### Removed
 
 ### Fixed
 
+## v0.5.1
+
+### Changed
+
+Changed the Pick and Place Demo's topic from SourceDestination to SourceDestination_input
```
Comment (sdiao) [inline CHANGELOG.md:27]: We can remove this line, since it went out in 0.5.0. Instead, we can add "Added Navigation 2 SLAM Example". what do you think?

Comment (mrpropellers) [reply CHANGELOG.md]: Sounds good - I'm assuming everything else in the ChangeLog is post-v0.5.0 changes?

File: README.md [hunk 2]
```diff
@@ -26,13 +26,18 @@ Brief steps on installing the Unity Robotics packages.
 
 A complete end-to-end demonstration, including how to set up the Unity environment, how to import a robot from URDF, and how to set up two-way communication with ROS for control.
 
-### [**New!**] [Object Pose Estimation Demo](https://github.com/Unity-Technologies/Robotics-Object-Pose-Estimation)
+### [Object Pose Estimation Demo](https://github.com/Unity-Technologies/Robotics-Object-Pose-Estimation)
```
Comment (sarah-gibson) [inline README.md:29]: We have not been consistent or precise in the past with our usage of "demo" vs. "tutorial" vs. "example". Reading this list of "demo, demo, demo, demo, example" makes it obvious. What do you think about changing "demo" in these titles to "Example" or "Tutorial" (which would be case-dependent). The downside is that the name here wouldn't match the repo name and all the text in the source repos.

**Review by sarah-gibson — APPROVED** [review]

**Review by sdiao — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mrpropellers) [reply README.md:29]: Well in OPE case it already doesn't match so I don't see much issue with changing it to "Example" or "Tutorial." I feel like both OPE and PNP would be better described as "Tutorials"

**Pull request merged**